backend/backend/api/routes.py

```diff
@@ -130,12 +130,12 @@ async def read_root():
     </style>
```

Comment:
Consider renaming the buttons from \u2018Test\u2019 and \u2018Tell Joke\u2019 to \u2016Research\u2017 and \u2016Get Joke\u2017, respectively, for clarity and consistency with the rest of the UI.